build.gradle

```diff
@@ -94,10 +99,17 @@ dependencies {
     compile 'net.lingala.zip4j:zip4j:2.3.1'*/
 }
 
+ext {
```

Comment:
```suggestion
def isDev = project.hasProperty("dev")
```
?

Reply:
I think it's best to tend to the same syntax/property set across our different gradle projects.
It will be easier to maintain and easier to eventually gather shared configurations in a gradle plugin in the future.

Reply:
Ok let's use def for this one and see if we need to change it in the future.